pkg/utils/gardener/seed_test.go

```diff
@@ -175,4 +178,44 @@ var _ = Describe("utils", func() {
 			Expect(err).NotTo(HaveOccurred())
 		})
 	})
+
+	Describe("#SeedIsGarden", func() {
+		var (
+			ctx        context.Context
+			mockClient *mockclient.MockClient
```

Comment:
Do not use a mock client

Reply:
Why shouldn't I use a mock client? I actually need it to mock the `NoResourceMatchError` error. I'm not aware of any way to do the same with the fake client.